### ISSUE TITLE
Ability to shuffle non-morphing paths

### ISSUE THREAD

**Issue by nickbutcher** [alexjlockwood/ShapeShifter#219]
I'm currently working on an animation involving trimming stroked paths. I'd like to be able to use the tooling for shuffling point order that is only available in the path morph editor.

**Comment by alexjlockwood** [comment]
Do you mean you want the ability to reverse/shift points in the main editor itself? So that, for example, you can clean up certain trim stroke animations (i.e. instead of animating from "0.4653945", shift the points so that you can animate from "0" instead)?

**Comment by nickbutcher** [comment]
Yes exactly. Animating `trimOffset` doesn't seem to like wrapping around (not sure if this is just in ShapeShifter or also in `AVD`) so having the ability to re-order the path would be helpful.

**Comment by alexjlockwood** [comment]
Oh, so you are actually having issues with writing the AVD itself? I would have thought that the order of the points wouldn't affect the trim path offset... do you have a simple example I could take a look at?
